week1/5.py

```diff
@@ -0,0 +1,10 @@
+""" Write a Python program to calculate body mass index.
+formula : BMI = weight(kg)/height*height(m) """
+
+weight = float(input("Input your weight in Kilogram: "))
+height = float(input("Input your height in meters: "))
+print("Your body mass index is: ", round(weight / (height * height), 2))
```

Comment:
`height * height` can be written in Python with the power to operator. Try to google it and improve his line